### DISCUSSION TITLE
ChatGPT4什么时候支持

### DISCUSSION THREAD

**Discussion by zixiaog** [ChatGPTNextWeb/NextChat#797] [General]
ChatGPT4什么时候支持

**Comment by Corwin006** [comment]
首先你要有使用OPENAI api GPT4接口的权限，你排队拿到了吗？
我申请好久了，没回应😂

**Comment by kanrun** [comment]
我也是，之前可以用的，后来就不能用了，不会刚开始是gpt4试用吧

**Comment by zhpfe** [comment]
我已升级pro，为啥还是用不了gpt4,官方app和官网可以使用

**Reply by Drmiaomiao** [reply]
api和官网账号是两码事
